server/start.go

```diff
@@ -171,6 +186,8 @@ is performed. Note, when enabled, gRPC will also be automatically enabled.
 	cmd.Flags().Bool(FlagInterBlockCache, true, "Enable inter-block caching")
 	cmd.Flags().String(flagCPUProfile, "", "Enable CPU profiling and write to the provided file")
 	cmd.Flags().Bool(FlagTrace, false, "Provide full stack traces for errors in ABCI Log")
+	// TODO(bweng):: disable by default for mainnet
+	cmd.Flags().Bool(FlagProfile, true, "Enable Profiling in the application")
```

Comment:
I think here we can just set default to false, and in our sei-infra, we can modify the systemd service to turn it on by default. wdyt?

Reply:
Yeah sounds good